packages/devextreme/js/__internal/ui/chat/messagebox.ts

```diff
@@ -90,6 +91,8 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
         const shouldButtonBeDisabled = !this._isValuableTextEntered();
 
         this._toggleButtonDisableState(shouldButtonBeDisabled);
+
+        triggerResizeEvent(this.$element().parent());
```

Comment:
What it's necessary for?

Reply:
When expanding the message box area, we need to update the scrollbar size when using the simulated strategy. Currently, the standard mechanism with the 'dxresize' trigger to call Chat's _dimensionChanged to notify Scrollable that it should be updated.

Reply:
Hm, shouldn't be there a test for this scenario?

And maybe we can make this solution more direct through direct call of a `messageList.update()`? I really don't like we trigger additional resize events for the whole chat

Reply:
` I really don't like we trigger additional resize events for the whole chat `

Yes, I understand, but it looks better to add a resize observer to the message list container. 

`Hm, shouldn't be there a test for this scenario?`
Yes, it should. I would to approve the solution and after this add test. Allow me to do it in separate PR.

Reply:
>  but it looks better to add a resize observer to the message list container.
Sounds good to me

Reply:
Well, let's add a test and rework the solution using resizeObserver in a neighbor PR